lib/src/main/java/org/team100/lib/localization/NotePosition24ArrayListener.java

```diff
@@ -73,18 +80,25 @@ public Optional<Double> getX() {
      * @return The y position in the camera in pixels, 0 should be the bottom of the
      *         screen
      */
-    public Optional<Double> getY() {
+    private Optional<Double> getY() {
```

Comment:
what does this represent?  the comment says something about pixels.  but it's used below as pitch.  what is this?